### ISSUE TITLE
[BUG] v3 /uploadListingImage - 'Either a valid image file or a listing_image_id must be provided.' 

### ISSUE THREAD

**Issue by tanishqandmac** [etsy/open-api#180]
**Describe the bug**
It seems like no matter what image we upload via "uploadListingImage" point  (v3) it always throw error.

**To Reproduce**
```
import requests
url = https://openapi.etsy.com/v3/application/shops/XXXXXX/listings/XXXXXX/images
data = { 'image':open('test.png', "rb")}
headers = {'x-api-key': 'XXXXXXX', 'Content-Type': 'application/x-www-form-urlencoded', 'Authorization': 'Bearer 12345.XXXXXXX'}

requests.post(url = url, data=data,  headers=headers)
```

**Expected behavior**
'Either a valid image file or a listing_image_id must be provided.'

**Additional context**
This seems to work fine if existing listing_image_id is provided rather than image. I also tried to add mime-type to image and also converted php code to python from this url - https://developers.etsy.com/documentation/tutorials/listings but it just doesn't seems to be working at all. My Etsy Request UUID - "Eu3c-xHITfywCE3E0OeE80gmEs20"

Has anyone found any workaround for this? Any help would be greatly appreciated.

**Comment by etsyachristensen** [comment]
@tanishqandmac Were you able to figure out the issue? I noticed you closed the ticket so we want to make sure you are good to go. If you don't mind, could you comment as to what you found that fixed the issue? It may help other users out there as well. Thanks!

**Comment by stefanus-ricky** [comment]
Solved at #233 
Try to remove the content-type
